data/data/bootstrap/systemd/units/release-image.service.template

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Downloads the OpenShift Release Image
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/bin/podman pull "{{.ReleaseImage}}"
```

Comment:
podman pulls have known to fail due to various reasons. `oneshot` means that if we fail to pull release image because of transient failures, our bootstrapping will fail making it let resilient.

Reply:
Sure, that makes sense, I updated this to follow the same pattern as bootkube to restart on failure + `set -e` + the conditional gating on a done file.